ui/js/services/jobfilters.js

```diff
@@ -407,15 +407,25 @@ treeherder.factory('thJobFilters', [
          */
         function getFieldFiltersArray() {
             const fieldFilters = [];
+            const clopt = thClassificationTypes.classificationOptions;
 
             _.each($location.search(), function (values, fieldName) {
                 if (_isFieldFilter(fieldName)) {
-                    const valArr = _toArray(values);
+                    var valArr = _toArray(values);
```

Comment:
I'm not sure why this was a `const`; I think it's always changing.

Reply:
It doesn't change within this scope.  It gets re-created each time we loop through this ``_.each`` item.  So ``const`` is still appropriate here.